src/functions/compact.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = function compact(array) {
+  if (!Array.isArray(array)) {
+    return [];
+  }
+
+  return array.filter(function(element) {
```

Comment:
What about this?

```suggestion
  return array.filter(Boolean);
```

Reply:
I wasn't too sure about the compatibility of using Boolean without new

Reply:
You were already using `Boolean` without new :)

```return Boolean(element);```

Anyway, I guess it's okay to do so: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean